src/rule-utils/__tests__/index.ts

```diff
@@ -319,3 +321,50 @@ describe('objectHash', () => {
     )
   })
 })
+
+describe('styleEq', () => {
+  test('can test style objects for equality', async (): Promise<void> => {
+    expect.assertions(2)
+    const { utils } = await buildUtils(
+      './empty.sketch',
+      createAssistantDefinition({
+        config: createAssistantConfig({ rules: { foo: { active: true } } }),
+        rules: [createRule({ name: 'foo' })],
+      }),
+      'foo',
+    )
+
+    const style1 = style
```

Comment:
Instead of hard coding these style objects, how about grabbing them out of a real Sketch file fixture? You can load a Sketch file in a test using the `fromFile` function.

```
const file = await fromFile(resolve(__dirname, './fixture.sketch'))
const style = file.contents.document.pages[0].layers[0].style
```

Reply:
Awesome! working on this 👍